tests/console/consoletest_setup.pm

```diff
@@ -36,6 +36,14 @@ sub run {
     script_run "ip addr show";
     save_screenshot;
 
+    # Stop serial-getty on serial console to avoid serial output pollution with login prompt
+    # Mask and stop only if SERIALDEV is defined or is qemu backend 
```

Comment:
Trailing whitespace (minor nit).

Reply:
well, not just minor as that's the travis fail :)

Reply:
Fixed. Thanks!